src/mappings/points/handlePoints.ts

```diff
@@ -0,0 +1,251 @@
+import { Ctx, BlockHeader_ } from '../../processor'
+import { Points } from '../../model'
+import { BackstopPool, SwapPool } from '../../model'
+
+import { Big } from 'big.js'
+
+import { getBackstopPoolLPPrice, getSwapPoolLPPrice } from './helpers'
+import fs from 'fs'
+
+type address = string
+type BackstopPoolId = string
+type SwapPoolId = string
+
+const OUTPUT_CSV_PATH = 'points.csv'
+const DUMP_BLOCK_HEIGHT = 3795124 // whenever the campaign finishes.
+
+// global vars to count points
+export const pointsCount = new Map<address, Big>()
```

Comment:
We need points to be a `Big`. The points per block obtained by the current equation can be very small (10**-7)